python/GafferUI/VectorDataWidget.py

```diff
@@ -487,7 +501,8 @@ def __emitDataChangedSignal( self, *unusedArgs ) :
 	def __contextMenu( self, pos ) :
 
 		# build the menu and pop it up
-		m = self._contextMenuDefinition( self.__selectedRows() )
+		m = IECore.MenuDefinition()
```

Comment:
If we just leave this line alone, we can drop all the changes to `_contextMenuDefinition()` and drop `__dataMenu()` too.

Reply:
Fixed and squashed into 71a0621e4ce0494c03be56e1315fa5a62a9ff1f2